nextcord/message.py

```diff
@@ -686,8 +686,9 @@ def __init__(
         self.id: int = int(data['id'])
         self.webhook_id: Optional[int] = utils._get_as_snowflake(data, 'webhook_id')
         self.reactions: List[Reaction] = [Reaction(message=self, data=d) for d in data.get('reactions', [])]
-        self.attachments: List[Attachment] = [Attachment(data=a, state=self._state) for a in data['attachments']]
-        self.embeds: List[Embed] = [Embed.from_dict(a) for a in data['embeds']]
+        self.attachments: List[Attachment] = \
+            [Attachment(data=a, state=self._state) for a in data.get('attachments', [])]
+        self.embeds: List[Embed] = [Embed.from_dict(a) for a in data.get('embeds', [])]
```

Comment:
Formatting suggestion

```suggestion
        self.attachments: List[Attachment] = [
            Attachment(data=a, state=self._state) for a in data.get("attachments", [])
        ]
        self.embeds: List[Embed] = [Embed.from_dict(a) for a in data.get("embeds", [])]
```